modules/utxo-lib/test/integration_local_rpc/generate/main.ts

```diff
@@ -36,7 +36,7 @@ async function initBlockchain(rpc: RpcClient, network: Network): Promise<void> {
   }
 }
 
-function toRegtestAddress(network: Network, scriptType: ScriptType, script: Buffer): string {
+function toRegtestAddress(network: { bech32?: string }, scriptType: ScriptType, script: Buffer): string {
```

Comment:
Is the `bech32` variable being used anywhere?

Reply:
yep it is passed to `fromOutputScript` later